code/Model/Submission/SubmittedFileField.php

```diff
@@ -36,10 +36,8 @@ class SubmittedFileField extends SubmittedFormField
     /**
      * Return the value of this field for inclusion into things such as
      * reports.
-     *
-     * @return string
      */
-    public function getFormattedValue()
+    public function getFormattedValue(): ?DBField
```

Comment:
While this method has the same name as FormField::getFormattedValue, it's a fundamentally different method

Putting strong typing on this is effectively unrelated refactoring